web_src/css/base.css

```diff
@@ -1060,10 +1060,15 @@ a.label,
 }
 
 .ui.pointing.dropdown > .menu:not(.hidden)::after {
-  background: var(--color-box-body);
+  background: var(--color-menu);
   box-shadow: -1px -1px 0 0 var(--color-secondary);
 }
 
+.ui.pointing.upward.dropdown .menu::after,
+.ui.top.pointing.upward.dropdown .menu::after {
```

Comment:
Curious, what are these style for?

And what doest `.top.pointing.upward` mean?